bedrock/firefox/views.py

```diff
@@ -570,6 +570,8 @@ def download_thanks(request):
         template = 'firefox/new/wait-face/scene2.html'
     elif lang_file_is_active('firefox/new/reggiewatts', locale) and experience == 'reggiewatts':
         template = 'firefox/new/reggie-watts/scene2.html'
+    elif lang_file_is_active('firefox/new/scene2_email', locale):
```

Comment:
It looks like we'll need to move this further down this crazy piece of spagetti, becuse both `de` and `en-US` are in the list of locales. It should be the final `else` on line 601:

```python
elif lang_file_is_active('firefox/new/scene2_email', locale):
    template = 'firefox/new/scene2_email.html'
else:
    template = 'firefox/new/scene2.html'
```
You'll then need to update some failing tests because the template name has changed. You can see the failing tests locally by running `docker-compose run test py.test bedrock/firefox`.